filter/timestamp_range_test.go

```diff
@@ -19,14 +20,14 @@ func TestTimestampRange(t *testing.T) {
 			name:      "valid",
 			ts:        1580981641, // 2020-02-06 09:34:01
 			startDate: "2019-02-06 09:34:01",
-			endDate:   "2022-02-06 09:34:01",
+			endDate:   "2122-02-06 09:34:01",
```

Comment:
Why is this change necessary?

Reply:
it's not, I thought about "now" I then I updated to a date more in the future, but I realize we use `ts` for the test